hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/index/bloom/SparkHoodieBloomIndex.java

```diff
@@ -252,11 +252,9 @@ public boolean isImplicitWithStorage() {
    * Make sure the parallelism is atleast the groupby parallelism for tagging location
    */
   JavaPairRDD<HoodieKey, HoodieRecordLocation> findMatchingFilesForRecordKeys(
-      final Map<String, List<BloomIndexFileInfo>> partitionToFileIndexInfo,
-      JavaPairRDD<String, String> partitionRecordKeyPairRDD, int shuffleParallelism, HoodieTable hoodieTable,
+      JavaRDD<Tuple2<String, HoodieKey>> fileComparisonsRDD,
+      int shuffleParallelism, HoodieTable hoodieTable,
```

Comment:
Maybe we can move `fileGroupToComparisons ` into this line or break the `hoodieTable ` to the next line. It would be better?

Reply:
Thanks, agree ~